android/titanium/src/java/org/appcelerator/titanium/TiRootActivity.java

```diff
@@ -71,6 +72,31 @@ public String getUrl()
 	@Override
 	protected void onCreate(Bundle savedInstanceState)
 	{
+		TiApplication tiApp = getTiApp();
+		Intent intent = getIntent();
+		TiRootActivity rootActivity = tiApp.getRootActivity();
+
+		if (intent != null) {
+			if (rootActivity != null) {
+				rootActivity.setIntent(intent);
+			}
+			if (tiApp.intentFilterNewTask() &&
+				intent.getAction().equals(Intent.ACTION_VIEW) &&
```

Comment:
Launching the app when building causes a crash as intent.getAction() = null. 

Launching the app manually is successful as action = Intent.ACTION_MAIN.